src/test/java/org/generationcp/commons/service/impl/GermplasmExportServiceImplTest.java

```diff
@@ -263,12 +263,6 @@ public void testWriteColumnValuesWithEmptyInventoryAmountValues() {
 			rowCount++;
 		}
 	}
-	
-	@Test
-	public void testGenerateExcelFileForSingleSheet() throws IOException {
-		this.germplasmExportService.generateExcelFileForSingleSheet(this.exportRows, this.columnsHeaders,
-				GermplasmExportTestHelper.TEST_FILE_NAME, this.sheetName);
-	}
 
 	@Test
```

Comment:
`testCreateWorkbookForSingleSheet` can be removed too?

Reply:
Done